src/towncrier/newsfragments/669.misc.rst

```diff
@@ -0,0 +1 @@
+Add your info here
```

Comment:
Ehm :)

Reply:
Perhaps we should teach `towncrier create` what `.misc` means. :upside_down_face:

Reply:
Okay, I blanked it out.